project-base/app/config/packages/dev/monolog.yaml

```diff
@@ -1,14 +1,14 @@
 monolog:
     handlers:
-        main:
-            type: "null"
-            level: emergency
-            excluded_http_codes: []
         log_stream:
             type: stream
             path: "%shopsys.log_stream%"
             level: notice
             formatter: "monolog.formatter.line"
+        main:
```

Comment:
Loggers are defined as a stack here and are processed in the defined order – see https://symfony.com/doc/current/logging.html#handlers-writing-logs-to-different-locations

This may change the behavior